### PR TITLE
Change originalLanguage to ISO639-1 standard

### PR DESCRIPTION
**Pull request by schneidermic0**

### CODE REVIEW

**Comment by larshp** [comment]
need to update https://github.com/SAP/abap-file-formats/blob/main/docs/json.md#original-language

**Comment by albertmink** [comment]
Closes #34

**Comment by larshp** [comment]
suggest adding a regular expression in the schema validation, to check its lower case(are numbers allowed?)

**Comment by schneidermic0** [comment]
> suggest adding a regular expression in the schema validation, to check its lower case(are numbers allowed?)

I can check whether I can add a `pattern` (like `[a-z]{2}`) to the language field. As far as I can see numbers are not allowed

**Comment by schneidermic0** [comment]
I have added the following attributes for language fields

```
"minLength": 2,
"maxLength": 2,
"pattern": "^[a-z]+$"
```

**Comment by schneidermic0** [comment]
BTW - I have also fixed the pattern for numerics. It was `[0-9]*`, before. For this pattern any string was accepted because the start (`^`) and the end (`$`) of the string where missing. `*` had to be replaced by `+` so that at least one number is available.
Therefore, it is `$[0-9]+$`, now

File: file-formats/chko/chko-v1.json [hunk 1]
```diff
@@ -108,7 +110,7 @@
             "maxLength": 255
           },
           "hidden": {
-            "title": "Is Hidden",
+            "title": "Is hidden",
```
Comment (wurzka) [inline file-formats/chko/chko-v1.json:113]: Is this an intended change to the title?

**Review by wurzka — CHANGES_REQUESTED** [review]
In the json.md file, there is an example with a language. There, the minLength and pattern has to be added, too.

File: file-formats/chko/chko-v1.json (hunk 1 above)
Comment (schneidermic0) [reply file-formats/chko/chko-v1.json:113]: No, it's not intended. Thanks. I'll fix it.

Comment (schneidermic0) [reply file-formats/chko/chko-v1.json]: Fixed

**Review by wurzka — CHANGES_REQUESTED** [review]
I guess you missed my comment about the json.md file above. Can you add the additional fields in the [example ](https://github.com/SAP/abap-file-formats/blob/languageHandling/docs/json.md#example)json schema there?

**Comment by schneidermic0** [comment]
@wurzka Yes, sorry. I missed your comment.

**Review by wurzka — APPROVED** [review]
Thanks, LGTM :)

**Pull request merged**